### PR TITLE
align internship programs cards to center

### PR DESCRIPTION
**Pull request by sketchomania**
Signed-off-by: vaibhav kushwaha <vaibhavvast6529@gmail.com>

**Description**

This PR fixes alignment of internship programs cards to center

**Notes for Reviewers**
- [issue 3468](https://github.com/layer5io/layer5/issues/3468)

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by legendarykamal** [comment]
@Nikhil-Ladha please approve the workflow.

**Comment by l5io** [comment]
🚀 Preview for commit f13edd1a908f7dcf79046cbc4d225cb6c938a7c1 at: https://6395a9ebce26906524362ebc--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
While you are there can you please add another change to the `ProgramsPageWrapper`?
That being remove the `margin-top: 2rem` CSS at line: 4 of the stylesheet. It is redundant.

**Comment by sketchomania** [comment]
Done ✅  @Nikhil-Ladha 
removed `margin-top:  -2rem`

**Comment by Nikhil-Ladha** [comment]
Please fix the DCO by following the steps mentioned [here](https://github.com/layer5io/layer5/pull/3516/checks?check_run_id=10027157951)

**Comment by leecalcote** [comment]
Thanks, @sketchomania. 👏

**Comment by eeshaanSA** [comment]
Why is the build preview failing here? @sketchomania

**Comment by sketchomania** [comment]
I am not sure about this, it was working locally and the error path  `error Building static HTML failed for path "/cloud-native-management/meshery/integrations/consul"` 
I havn't changed anything in this path.

Can I create a create a pull request from new branch this will also help in tracking what going wrong.
@eeshaanSA

**Comment by legendarykamal** [comment]
no @sketchomania wait .I think I know the fault.

**Comment by legendarykamal** [comment]
Somethings wrong here @sketchomania with the `integrationIcon` 

image

**Comment by sketchomania** [comment]
Can anything be done here @legendarykamal 
I didn't modified anything related to `integrationIcon`

**Comment by legendarykamal** [comment]
> Can anything be done here @legendarykamal I didn't modified anything related to `integrationIcon`

yes try syncing it with the main repo , you actually added other commits to ur repo . Hence , the PR is this way. 
Main repo is working nice . Sync with it .

**Comment by sketchomania** [comment]
I tried syncing with main repo, still getting the same error  @legendarykamal

**Comment by sketchomania** [comment]
I tried syncing with the main repo, still getting the same error
@legendarykamal 
It is working on this new branch https://github.com/sketchomania/layer5/tree/align_internship_programs_cards

**Review by legendarykamal — APPROVED** [review]
LGTM now 👍

File: src/collections/integrations/aerospike-kubernetes-operator/aerospike-kubernetes-operator.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ A Helm chart for Aerospike Kubernetes Operator.
+
+
+</p>
+<p>
+ Collaboratively and visually diagram your cloud native infrastructure with GitOps-style pipeline integration. Design, test, and manage configuration your Kubernetes-based, containerized applications as a visual topology.
+</p>
+<p>
+ Looking for best practice cloud native design and deployment best practices? Choose from thousands of pre-built components in MeshMap. Choose from hundreds of ready-made design patterns by importing templates from Meshery Catalog or use our low code designer, MeshMap, to create and deploy your own cloud native infrastructure designs.
+</p>
```
Comment (Nikhil-Ladha) [inline src/collections/integrations/aerospike-kubernetes-operator/aerospike-kubernetes-operator.mdx:30]: This shouldn't be a part of this PR.

Comment (sketchomania) [reply src/collections/integrations/aerospike-kubernetes-operator/aerospike-kubernetes-operator.mdx:30]: How can I fix this ?
and
Is this branch is good https://github.com/sketchomania/layer5/tree/align_internship_programs_cards

Comment (Nikhil-Ladha) [reply src/collections/integrations/aerospike-kubernetes-operator/aerospike-kubernetes-operator.mdx:30]: Try removing the folder, and rebasing the branch.

**Comment by sketchomania** [comment]
I have already mentioned this separate branch is working without any conflict
 https://github.com/sketchomania/layer5/tree/align_internship_programs_cards
 have anyone checked it ?

**Comment by Nikhil-Ladha** [comment]
> I have already mentioned this separate branch is working without any conflict https://github.com/sketchomania/layer5/tree/align_internship_programs_cards have anyone checked it ?

Well, I shouldn't be the one coming back and asking for it. You should be the one 🙂.
Secondly, comparing the branch is too much effort which I am not willing to put, creating a PR is an easier job that you can do from the new branch that you believe is correct. Then, wd could review and move ahead with it accordingly.

**Comment by sketchomania** [comment]
Thanks @Nikhil-Ladha   got your point

here is the new pull request 
https://github.com/layer5io/layer5/pull/3654

**Comment by sketchomania** [comment]
I got unassigned from this issue

**Comment by Nikhil-Ladha** [comment]
Yeah, because there wasn't any update from your end for a long time and a new contributor showed interest and made a quick PR.
You can close the the new PR. This wasn't an issue that required to be extended for more than 1 month 🙂

**Comment by sketchomania** [comment]
closed the new PR.
I think there was a confusion at my side I will take care this next time 🙂

**Pull request closed without merging**